README.md

```diff
@@ -91,6 +91,42 @@ In VS Code, open up the Marketplace tab. Then click on the 3 horizontal dots ico
 
 ![VS Code Install Extension](./media/vscode-install-extension.png)
 
+## Local Development
+
+### Install esbuild
+
+### Building the Extension
+
+To build the extension, you'll need `esbuild` installed:
+
+```
+npm install -g esbuild
```

Comment:
This is already listed in the `devDependencies` so an `npm install` should automatically download this tool for you. It should work with the npm scripts out of the box as long as you have `./node_modules/.bin` in your PATH. The big benefit to this is that we can lock the version of the tool for the project. Could we update this to reflect?

Reply:
> It should work with the npm scripts out of the box as long as you have ./node_modules/.bin in your PATH.

This was definitely not on my PATH. Is this something that's expected to do when you setup node? I don't think I've ever run into anything explaining to do this, which makes me think there are probably a not insignificant number of people who would be in the same boat as me where running `npm install` would not result in `esbuild` being available. I guess just replace the portion saying to install `esbuild` globally with something saying to add `./node_modules/.bin` to the path?

Reply:
Went ahead and made that change and force-pushed it.

Reply:
Ah, you're right! I just realized that I've got a custom script to specifically add my most common repos' `node_modules/.bin` to my path. It looks like the [modern recommendation](https://stackoverflow.com/questions/9679932/how-to-use-executables-from-a-package-installed-locally-in-node-modules) would be to run `npx esbuild ...` in our scripts, which will attempt to use the locally installed esbuild or install it fresh otherwise.

```diff
diff --git a/package.json b/package.json
index 7cab245..339af36 100644
--- a/package.json
+++ b/package.json
@@ -139,8 +139,8 @@
     "typecheck": "tsc --version && tsc",
     "lint": "eslint \"src/**/*.ts\"",
     "prettier": "prettier . --check",
-    "compile-dev": "esbuild `find ./src -not \\( -path ./src/test -prune \\) \\( -name '*.ts' \\)` --format=cjs --platform=node --sourcemap --outdir=./out",
-    "compile-prod": "esbuild ./src/extension.ts --format=cjs --platform=node --minify --bundle --external:vscode --outfile=out/extension.js",
+    "compile-dev": "npx esbuild `find ./src -not \\( -path ./src/test -prune \\) \\( -name '*.ts' \\)` --format=cjs --platform=node --sourcemap --outdir=./out",
+    "compile-prod": "npx esbuild ./src/extension.ts --format=cjs --platform=node --minify --bundle --external:vscode --outfile=out/extension.js",
     "watch": "concurrently -i -c green,blue, -n check,build 'npm run check' 'npm run compile-dev -- --watch'",
     "test": "npm run compile-dev && cd ./src/test && doppler configure set token '$DOPPLER_TOKEN' --silent && doppler setup -p vscode -c test --scope=. --silent && node ./runTest.js",
     "test-local": "DOPPLER_TOKEN=$(doppler configure get token --plain) npm run test",


```

Reply:
Do we want to do that vs ask for a `PATH` update? Adding the relative `./node_modules/.bin` path to `PATH` does seem to work – it just checks in whatever directory you happen to be in.

Reply:
Ah, I see. Your link discusses why you don't necessarily want to do that. I'll push up a change to swap this over to `npx`.